src/com/phonebookservice/util/ErrorMessages.java

```diff
@@ -0,0 +1,16 @@
+package com.phonebookservice.util;
+
+/**
+ * Error messages.
+ *
+ * @author Marwa Saleh
+ */
+public abstract class ErrorMessages {
+
+    public static final String ERROR_DATABASE_NOT_FOUND //
+            = "ERROR_DATABASE_NOT_FOUND";
```

Comment:
Message should be readable for user. 

for example : "Database object cannot be null"

Reply:
same in bellow messages